rosa_hcp/rosa-hcp-shared-vpc-config.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-hcp-shared-vpc-config"]
+= Configuring a shared VPC for ROSA with HCP clusters
+:context: rosa-shared-vpc-config
+
+toc::[]
+
+You can create {hcp-title-first} clusters in shared, centrally-managed AWS virtual private clouds (VPCs).
+
+[NOTE]
+====
+This process requires *two separate* AWS accounts that belong to the same AWS organization. One account functions as the VPC-owning AWS account (*VPC Owner*), while the other account creates the cluster in the cluster-creating AWS account (*Cluster Creator*).
+====
+
+image::522-shared-vpc-overview.png[]
+
+* The hosted zones can be created in either the centrally-managed VPC account or in the workload account in which the cluster is deployed.
+
+.Prerequisites for the *VPC Owner*
+* You have an AWS account with the proper permissions to create roles and share resources.
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
+* Both AWS accounts belong to the same AWS organization.
+* You link:https://docs.aws.amazon.com/ram/latest/userguide/getting-started-sharing.html#getting-started-sharing-orgs[enabled resource sharing] from the management account for your organization.
+* You have access to an AWS entrypoint such as the link:https://signin.aws.amazon.com[AWS console] or the link:https://aws.amazon.com/cli/[AWS command-line interface] (CLI).
+
+.Prerequisites for the *Cluster Creator*
+* You installed the link:https://console.redhat.com/openshift/downloads#tool-rosa[ROSA CLI (`rosa`)] 1.2.49 or later.
+* You created all of the required link:https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.html[ROSA account roles] for creating a cluster.
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
+* Both AWS accounts belong to the same AWS organization.
+
+[NOTE]
+====
+Installing a cluster in a shared VPC is supported only for OpenShift 4.17.9 and later, 4.13.10 and later, and all future 4.y-streams.
```

Comment:
We should change this to be 4.17.9 and later only. 4.13 and later was only for ROSA Classic.